Cargo.toml

```diff
@@ -2,6 +2,10 @@
 # Make sure any excluded crates are still clipped in our CI. Also, make sure they use the same Clippy and rustc lints we specify in this file down below.
 exclude = ["nomos-core/proof_statements", "nomos-core/risc0_proofs"]
 members = [
+  "circuits/pol",
```

Comment:
Should we also add them to the workspace dependencies, down below? So other crates can import them, if we need? If not, perhaps we do not need that now.